common/cluster/archival_status.go

```diff
@@ -29,11 +29,15 @@ type (
 	// ArchivalStatus represents the archival status of the cluster
 	ArchivalStatus int
```

Comment:
I keep the naming of the cluster level config as otherwise it will be referred as cluster.ClusterArchivalStatus outside the package.

Reply:
makes sense